### DISCUSSION TITLE
Limit of characters, words or letters?

### DISCUSSION THREAD

**Discussion by jhoanmartinez** [rany2/edge-tts#152] [General]
Grettings fellas, this package is awesome, is there any limit per request, limit of words, letters or characters? I was going through the code and i found a kind of api key, so before to drop a big text would like to know about this, big thanks.

**Comment by rany2** [comment]
There shouldn't be a limit you'd notice. The package automatically splits the text as is needed as to meet the text limit per request, so you shouldn't face an error over this. As for that API key, it's the API key that the Edge browser uses; not my own API key :)

**Reply by jhoanmartinez** [reply]
Thanks @rany2 for the quick reply.

**Reply by omega3** [reply]
@rany2 
> it's the API key that the Edge browser uses

Is it one and the same API for every Edge installation or every Edge installation produces its own individual API obtained from Edge installation? If the second is the case maybe it would make sense if every user pasted its own API to some configuration file, so as not to block other users.

**Reply by rany2** [reply]
@omega3 No, it does not generate an individual unique API key per installation. The API key is consistent for any Edge installation.
